.eslintrc.yml

```diff
@@ -0,0 +1,60 @@
+extends:
+- airbnb-base
+- next
+
+env:
+  browser: true
+  es2021: true
+  es6: true
+  jest: true
+
+globals:
+  google: readonly
+  module: readonly
+  process: readonly
+
+parserOptions:
+  ecmaVersion: 8
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+
+rules:
+  arrow-body-style: off
+  arrow-parens: off
+  func-names: warn
+  func-style: off
+  max-len: warn
+  no-bitwise: off
+  no-cond-assign:
+  - warn
+  - except-parens
+  no-confusing-arrow: off
+  no-console:
+  - warn
+  - allow:
+    - warn
+    - error
+  no-mixed-operators: off
+  no-nested-ternary: off
+  no-plusplus: warn
```

Comment:
I turned [this rule](https://eslint.org/docs/rules/no-plusplus) from an error into a warning because I think it's going to confuse people, because they've only ever used `i++` rather than `i += 1`. I agree with Airbnb that `i += 1` is better, but I don't want to unfairly punish people for what Hack Reactor taught them. You might even prefer to turn this one off entirely.